config/non-admin-controller_rbac/role.yaml

```diff
@@ -0,0 +1,43 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: non-admin-controller-role
+rules:
+- apiGroups:
+  - nac.oadp.openshift.io
+  resources:
+  - nonadminbackups
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - nac.oadp.openshift.io
+  resources:
+  - nonadminbackups/finalizers
+  verbs:
+  - update
+- apiGroups:
+  - nac.oadp.openshift.io
+  resources:
+  - nonadminbackups/status
+  verbs:
+  - get
+  - patch
+  - update
+- apiGroups:
+  - velero.io
+  resources:
+  - backups
```

Comment:
Where are the roles for restore?

Reply:
restore is not yet in.. just backup

Reply:
waiting this one merge to unblock this work for https://github.com/migtools/oadp-non-admin/pull/42